runtime/gc_base/StringTable.cpp

```diff
@@ -590,43 +600,43 @@ j9gc_createJavaLangString(J9VMThread *vmThread, U_8 *data, UDATA length, UDATA s
 			goto nomem;
 		}
 
-		if (IS_STRING_COMPRESSION_ENABLED_VM(vm)) {
-			if (J9_STR_UNICODE == (stringFlags & J9_STR_UNICODE)) {
-				unicodeLength = length / 2;
-				isCompressable = isUnicodeCompressable((U_16*)data, unicodeLength);
-			} else {
-				unicodeLength = getUnicodeLength(data, length, &isCompressable);
-			}
+      if (J9_STR_UNICODE == (stringFlags & J9_STR_UNICODE)) {
```

Comment:
There are some tabs vs spaces issues here

Reply:
Fixed in c86ff40